packages/aws-cdk/lib/api/aws-auth/awscli-compatible.ts

```diff
@@ -179,7 +179,7 @@ function getRegionFromIniFile(profile: string, data?: any) {
 function tryGetCACert(bundlePath?: string) {
   const path = bundlePath || caBundlePathFromEnvironment();
   if (path) {
-    debug('Using CA bundle path: %s', bundlePath);
+    debug('Using CA bundle path: %s', path);
```

Comment:
Codecov is failing because we don't have any unit test for this part, but the integ test in this PR covers precisely this branch.